README.md

```diff
@@ -93,12 +93,13 @@ The ACAP Runtime service provides the following APIs:
 
 - Machine learning API - An implementation of [TensorFlow Serving][tensorflow]. There
   are usage examples available for the Machine learning API written in
-  [Python][minimal-ml-inference] and [C++][object-detector-cpp].
+  [Python][minimal-ml-inference].
 - Parameter API - Provides gRPC read access to the parameters of an Axis device.
   There are usage examples available for the Parameter API written in
-  [Python][parameter-api-python] and [C++][paramter-api-cpp].
-- Video capture API - Enables capture of images from a camera. No usage examples
-  for this API exist yet.
+  [Python][parameter-api-python].
+- Video capture API - Enables capture of images from a camera.
+  There are usage examples available for the Parameter API written in
```

Comment:
I think you can remove this line - we don't list every example. Alternatively add it under the Examples section instead

Reply:
Do you mean the line:
"There are usage examples available for the Parameter API written in [Python][opencv-qr-decoder-python]." ?

Reply:
yes